services/sensors/builtin/builtin.go

```diff
@@ -8,22 +8,26 @@ import (
 	"github.com/pkg/errors"
 
 	"go.viam.com/rdk/components/sensor"
-	"go.viam.com/rdk/internal"
 	"go.viam.com/rdk/logging"
 	"go.viam.com/rdk/resource"
 	"go.viam.com/rdk/services/sensors"
 )
 
 func init() {
-	resource.RegisterDefaultService(sensors.API, resource.DefaultServiceModel, resource.Registration[sensors.Service, resource.NoNativeConfig]{
-		Constructor: NewBuiltIn,
-		// NOTE(erd): this ideally would be a matcher on all resources that
-		// contain a Readings proto method. However, these resource types
-		// are heterogeneous and do not actually have this proto method.
-		// We need to either 1. know in advance what kinds of resources are
-		// sensors 2. use a heuristic to detect it or 3. encode it in proto.
-		WeakDependencies: []internal.ResourceMatcher{internal.ComponentDependencyWildcardMatcher},
-	})
+	resource.RegisterDefaultService(
+		sensors.API,
+		resource.DefaultServiceModel,
+		resource.Registration[sensors.Service, resource.NoNativeConfig]{
+			Constructor: NewBuiltIn,
+			// NOTE(erd): this ideally would be a matcher on all resources that
+			// contain a Readings proto method. However, these resource types
+			// are heterogeneous and do not actually have this proto method.
+			// We need to either 1. know in advance what kinds of resources are
+			// sensors 2. use a heuristic to detect it or 3. encode it in proto.
+			// TODO: ^^^ ??
```

Comment:
You can probably remove this `TODO` and leave the note above. Doing what Eric describes here sounds... hard. I can't think of a way to do it besides `3`, which I, personally, don't really want to do (add an `IsSensor` field or method to everything). Worth leaving the note in in case anyone has a better idea.

Reply:
Actually I think I may have come up with something simple that can work.  Going to try it quickly